packages/client/hmi-client/src/workflow/ops/model-edit/tera-model-edit.vue

```diff
@@ -97,14 +110,33 @@ interface SaveOptions {
 	appendOutputPort?: boolean;
 }
 
-const kernelManager = new KernelSessionManager();
+const outputs = computed(() => {
+	if (!_.isEmpty(props.node.outputs)) {
+		return [
+			{
+				label: 'Select outputs to display in operator',
+				items: props.node.outputs
+			}
+		];
+	}
+	return [];
+});
+const selectedOutputId = ref<string>();
+const activeOutput = ref<WorkflowOutput<ModelEditOperationState> | null>(null);
 
+const kernelManager = new KernelSessionManager();
 const amr = ref<Model | null>(null);
+const contextInfo = { id: props.node.inputs[0].value?.[0] }; // context for jupyter-input
```

Comment:
Isn't this just a modelId?

Reply:
yep, I will define modelId and use it instead, that could allow us to use modelId elsewhere in this drilldown